README.md

```diff
@@ -17,9 +17,7 @@ As new standards and APIs become available `<model-viewer>` will be improved
 to take advantage of them. If possible, fallbacks and polyfills will be
 supported to provide a seamless development experience.
 
-See: [API](#api), [Demo](https://model-viewer.glitch.me),
-[Examples](https://googlewebcomponents.github.io/model-viewer/index.html),
-[Kanban](https://github.com/GoogleWebComponents/model-viewer/projects/1) (for work prioritization and progress)
+[Remixable demo](https://model-viewer.glitch.me) • [Documentation](https://googlewebcomponents.github.io/model-viewer/index.html) • [Kanban](https://github.com/GoogleWebComponents/model-viewer/projects/1) • [Fidelity test results](https://googlewebcomponents.github.io/model-viewer/test/fidelity/results-viewer.html)
```

Comment:
I strongly prefer Glitch to those tools because it puts an emphasis on education and beginner friendliness, while still offering really advanced features like the ability to run custom server-side business logic.

Reply:
(And visual version control even!)

Reply:
We have also received a lot of free social media magnification from them on Twitter.

Reply:
The main benefit of codepen or jsfiddle is that we'd be presenting the user the code next to the working example. Right now the user needs to click on "Remix this".

Reply:
That's certainly useful. Glitch does support this presentation mode [when it is embedded](https://glitch.com/embed/#!/embed/model-viewer?path=index.html?path=index.html:1:0). 

Personally I think we're better off with Glitch, given its emphasis on offering an educational environment that is accessible to beginners.

Reply:
Either way. As long as we're providing a fun sandbox for the user to easily play/learn.